bridge/client/app/_components/ktb-expandable-tile/ktb-expandable-tile.component.ts

```diff
@@ -97,8 +97,11 @@ export class KtbExpandableTileComponent {
     return this._disabled;
   }
   set disabled(value: boolean) {
-    if (this._disabled && this._selected) {
-      this._selected = false;
+    if (this._disabled !== value) {
+      this._disabled = value;
+      if (this._disabled) {
+        this.selected = false;
```

Comment:
Do we even need this property? Looks like it is never set to `true`

Reply:
Right. at some time we split this component into `ktb-selectable-tile` and this `ktb-expandable-tile`. For the KtbSelectableTile it does make sense, but here we can drop it.